setup.py

```diff
@@ -69,10 +69,12 @@
     install_requires=[
         "eth-abi>=2.0.0b4,<3.0.0",
         "eth-account>=0.2.1,<0.4.0",
+        "eth-hash[pycryptodome]>=0.2.0,<1.0.0",
         "eth-utils>=1.3.0,<2.0.0",
+        "ethpm>=0.1.4a10,<1",
         "hexbytes>=0.1.0,<1.0.0",
         "lru-dict>=1.1.6,<2.0.0",
-        "eth-hash[pycryptodome]>=0.2.0,<1.0.0",
+        "pytest-ethereum>=0.1.3a6,<1",
```

Comment:
Looks like this should be moved to the `tester` extra, instead of as a core dependency.

Reply:
Hmmm, maybe I'm missing something, but we use pytest-ethereum's `Deployer` in the `web3.pm` module. But, I would say that this use case of using the `Deployer` is not that critical at all, since it's a relatively straightforward deployment. So, I can definitely pull it out, in which case `pytest-ethereum` would only be needed in the tests and therefore the `tester` extra.

Reply:
Ah, I just did a Ctrl-f in the github diff, and the diff on `pm` was hidden.

At least semantically, I find it pretty surprising to use anything called `pytest-*` in the main deployed package. If `Deployer` is to be used in production code like this, then maybe it could be moved into a web3 `_utils` module.